lib/radiator_web/graphql/schema.ex

```diff
@@ -66,7 +66,8 @@ defmodule RadiatorWeb.GraphQL.Schema do
 
     @desc "Get one published episode"
     field :published_episode, :published_episode do
-      arg :id, non_null(:id)
+      arg :id, :id
+      arg :guid, :string
```

Comment:
I think I would prefer a more open endend query api, that also has guid as optional query element and returns a list instead.